### PR TITLE
webui: Update to Installation Guide | Improved labels & focus traffic to launcher

### PR DESCRIPTION
**Pull request by Yeldur**
# Update Installation Guide to push focus on Launcher whilst improving readability and labels

### Rename 'Other' tab to 'Launcher' 
_Tab previously labeled 'Other' is now renamed to 'Launcher' - This change is to make the purpose of the tab more explicit and user-friendly_
### Update Platform name from 'Steam' to 'Steam Workshop' 
_The label for the 'Steam' Platform in the installation guide is updated to 'Steam Workshop' - This clarifies the specific nature of Steam related content. From review, this label change appears localized and does not impact other areas of the codebase._
### Add Steam Icon to the 'Launcher' to improve visuals
_A Steam icon has been added next to the 'Launcher' tab. This visual addition is to indicate that Steam users can ALSO utilize the Launcher_
### Move 'Other' to sit in front of 'Steam' 
_The 'Launcher' tab (formerly named 'Other') has been moved to precede the 'Steam Workshop' tab (formerly named 'Steam'). This reordering aligns with our goal to emphasize the Launcher as the primary method for installation._
### Update en.yml text 
_Minor grammatical corrections to 'update' & 'help' keys_
### Update cn.yml text
_Minor changes to wording of 'help' key_
### Update Launcher link
_Updated launcher link from the moddb link to our own hosted link for download_
### Add installation guide video
_Added an installation guide in video form, to show users what they need to do in order to get cRPG downloaded & working in both Launcher & Steam Workshop format_

### CODE REVIEW

**Comment by or2e** [comment]
I propose to return to this issue after some minor tweaks https://github.com/namidaka/crpg/issues/230

**Comment by Yeldur** [comment]
No problems from me!

**Review by namidaka — DISMISSED** [review]

**Comment by or2e** [comment]
Don't forget to change the link to the video guide

**Review by or2e — DISMISSED** [review]

**Review by or2e — DISMISSED** [review]

File: src/WebUI/src/components/app/InstallationGuide.vue [hunk 1]
```diff
@@ -37,82 +37,83 @@ watch(
...
+            :value="PossibleValues.Steam"
+          >
+            <ol>
+              <i18n-t scope="global" keypath="installation.platform.steam.subscribe" tag="li">
+                <template #steamWorkshopsLink>
+                  <!-- prettier-ignore -->
+                  <a
+                    target="_blank"
+                    href="steam://openurl/https://steamcommunity.com/sharedfiles/filedetails/?id=2878356589"
+                  >Steam Workshops</a>
```
Comment (Muparadzi) [inline src/WebUI/src/components/app/InstallationGuide.vue:82]: Steam Workshop

File: src/WebUI/src/components/app/InstallationGuide.vue [hunk 2]
```diff
@@ -37,82 +37,83 @@ watch(
...
+              >
+                <template #launcherLink>
+                  <a target="_blank" href="https://c-rpg.eu/LauncherV3.exe">Launcher</a>
+                </template>
+              </i18n-t>
+              <li>{{ $t('installation.platform.other.startLauncher') }}</li>
+              <li>{{ $t('installation.common.bannerlordLauncher') }}</li>
+              <li>{{ $t('installation.common.multiplayerModsTab') }}</li>
+              <li>{{ $t('installation.common.activateMod') }}</li>
+              <li>{{ $t('installation.common.launchMultiplayerGame') }}</li>
```
Comment (Muparadzi) [inline src/WebUI/src/components/app/InstallationGuide.vue:65]: This isn't the process for using the launcher, after the game has updated, when they 'Launch CRPG' it should start bannerlord for them already in multiplayer, without having to worry about what mods to load & their load order,


image

**Review by Muparadzi — CHANGES_REQUESTED** [review]

File: src/WebUI/src/components/app/InstallationGuide.vue (hunk 2 above)
Comment (Yeldur) [reply src/WebUI/src/components/app/InstallationGuide.vue:65]: Quite right actually, I forgot that our Launcher process had changed around and didn't even think to touch that. I'll get this updated and will submit

File: src/WebUI/src/components/app/InstallationGuide.vue (hunk 1 above)
Comment (Yeldur) [reply src/WebUI/src/components/app/InstallationGuide.vue:82]: Will correct

File: src/WebUI/src/components/app/InstallationGuide.vue (hunk 2 above)
Comment (Yeldur) [reply src/WebUI/src/components/app/InstallationGuide.vue]: Updated now

File: src/WebUI/src/components/app/InstallationGuide.vue (hunk 1 above)
Comment (Yeldur) [reply src/WebUI/src/components/app/InstallationGuide.vue]: Corrected

**Comment by Yeldur** [comment]
orle will probably need to check the RU wording is correct

File: src/WebUI/locales/ru.yml [hunk 3]
```diff
@@ -11,21 +11,23 @@ platform:
...
+      activateMod: Активируйте мод cRPG
+      launchMultiplayerGame: Запустите игру в многопользовательском режиме!
     other:
       title: Лаунчер
       downloadLauncher: Загрузите наш {launcherLink}
       startLauncher: Откройте Launcher и подождите, пока мод загрузится
-      update: Для обновления мода достаточно перезапустить Launcher
-  help: Если у вас возникли трудности с установкой модификации, пожалуйста, свяжитесь с нами с помощью {discordLink}
-  watchVideoGuide: Видео инструкция
+      detectinstall: Нажмите "Определить установку", чтобы автоматически найти место установки, или задайте ero вручную c помощью "Изменить место".
```
Comment (or2e) [inline src/WebUI/locales/ru.yml:28]: Launcher has no translation, then its button titles do not need to be translated

Comment (Yeldur) [reply src/WebUI/locales/ru.yml:28]: Will update

Comment (Yeldur) [reply src/WebUI/locales/ru.yml]: Resolved in https://github.com/namidaka/crpg/pull/228/commits/1ae77cd78d174839aafb716ffdb7a35958fcee8c

**Review by or2e — APPROVED** [review]

**Review by Muparadzi — APPROVED** [review]

**Comment by or2e** [comment]
> orle will probably need to check the RU wording is correct

++

**Pull request merged**